c2corg_api/views/waypoint.py

```diff
@@ -28,7 +29,11 @@
 from functools import lru_cache
 from sqlalchemy.orm import joinedload, load_only
 from sqlalchemy.orm.util import aliased
-from sqlalchemy.sql.expression import and_
+from sqlalchemy.sql.elements import literal_column
+from sqlalchemy.sql.expression import and_, union
+
+# the number of routes that are included for waypoints
+NUM_ROUTES = 30
```

Comment:
I don't really see your problem. It is out of question to show 100 routes for a waypoint. So we show some (I think even 30 is too much), and if you want to see all you click on "Show all (594)" and you can browse all routes paginated.